.gitignore

```diff
@@ -6,7 +6,6 @@
 /Configuration/ComposerPackagesCommands.php
 /Documentation-GENERATED-temp
 /Resources/Private/ExtensionArtifacts/Libraries
-/Resources/Public/*
```

Comment:
I'd rather add an exception afterwards specifically for the icon. We do not want to have other, possibly generated, resources in VCS in the future.

Reply:
Done.